File: doc/alertas/ConsultaAvanceCreditos/App_Code/ewrglobal4.vb
Imports System
Imports System.Collections
Imports System.Collections.Generic
Imports System.Collections.Specialized
Imports System.Web
Imports System.Web.UI
Imports System.Data
Imports System.Data.Common
Imports System.Xml
Imports System.IO
Imports System.Security.Cryptography
Imports System.Text
Imports System.Text.RegularExpressions
Imports System.Drawing
Imports System.Drawing.Imaging
Imports System.Drawing.Drawing2D
Imports System.Reflection
Imports System.Net
Imports System.Net.Mail
Imports System.Net.Mime
Imports System.Globalization
Imports Microsoft.VisualBasic
Imports System.Data.SqlClient

'
' ASP.NET Report Maker 4 - Global
' (C) 2010 e.World Technology Ltd.
'
Public Partial Class AspNetReportMaker4_consultaAvanceCreditos
	Inherits System.Web.UI.Page

	'
	' Global variables
	'
	' Connection
	Public Conn As cConnection

	' Security
	Public Security As cAdvancedSecurity

	' Language
	Public Shared gsLanguage As String = ""

	' Export
	Public Shared gsExport As String = ""

	Public Shared gsExportFile As String = ""

	' Email
	Public Shared gsEmailSender As String = ""

	Public Shared gsEmailRecipient As String = ""

	Public Shared gsEmailCc As String = ""

	Public Shared gsEmailBcc As String = ""

	Public Shared gsEmailSubject As String = ""

	Public Shared gsEmailContent As String = ""

	Public Shared gsEmailContentType As String = ""

	Public Shared gsEmailErrNo As String = ""

	Public Shared gsEmailErrDesc As String = ""

	' Used by extended filter
	Public Shared gsFormError As String = ""

	' Debug message
	Public Shared gsDebugMsg As String = ""

	' Timer
	Public StartTimer As Long

	' Page Loading event
	Public Sub Page_Loading()

		'ew_Write("Page Loading")
	End Sub

	' Page Unloaded event
	Public Sub Page_Unloaded()

		'ew_Write("Page Unloaded")
	End Sub

	' Language
	Public ReportLanguage As crLanguage

	' Runtime variables	
	Public sSql As String = ""

	Public ChartObj As object	

	Public SqlSelect As String = ""

	Public SqlChartSelect As String = ""

	Public SqlChartBase As String = ""

	Public chartxml As String = ""
End Class

File: doc/alertas/EstadoAcademico/App_Code/ewrglobal4.cs
using System;
using System.Collections;
using System.Collections.Generic;
using System.Collections.Specialized;
using System.Web;
using System.Web.UI;
using System.Data;
using System.Data.Common;
using System.Xml;
using System.IO;
using System.Security.Cryptography;
using System.Text;
using System.Text.RegularExpressions;
using System.Drawing;
using System.Drawing.Imaging;
using System.Drawing.Drawing2D;
using System.Reflection;
using System.Net;
using System.Net.Mail;
using System.Net.Mime;
using System.Globalization;
using Microsoft.VisualBasic;
using System.Data.SqlClient;

//
// ASP.NET Report Maker 4 - Global
// (C) 2010 e.World Technology Ltd.
//
public partial class AspNetReportMaker4_project1: System.Web.UI.Page
{

	/**
	 * Global variables
	 */

	// Connection
	public cConnection Conn;

	// Language
	public static string gsLanguage = "";

	// Security
	public cAdvancedSecurity Security;

	// Export
	public static string gsExport = "";

	public static string gsExportFile = "";

	// Email
	public static string gsEmailSender = "";

	public static string gsEmailRecipient = "";

	public static string gsEmailCc = "";

	public static string gsEmailBcc = "";

	public static string gsEmailSubject = "";

	public static string gsEmailContent = "";

	public static string gsEmailContentType = "";

	public static string gsEmailErrNo = "";

	public static string gsEmailErrDesc = "";

	// Used by extended filter
	public static string gsFormError = "";

	// Debug message
	public static string gsDebugMsg = "";

	// Timer
	public long StartTimer;

	// Page Loading event
	public void Page_Loading() {

		//ew_Write("Page Loading");
	}

	// Page Unloaded event
	public void Page_Unloaded() {

		//ew_Write("Page Unloaded");
	}

	// Language
	public crLanguage ReportLanguage;

	// Runtime variables	
	public string sSql = "";

	public object ChartObj;	

	public string SqlSelect = "";

	public string SqlChartSelect = "";

	public string SqlChartBase = "";

	public string chartxml = "";
}
